src/utils/anim-utils.ts

```diff
@@ -69,6 +70,17 @@ export async function loadEncounterAnimAssets(startLoad?: boolean): Promise<void
   await loadAnimAssets(Array.from(encounterAnims.values()), startLoad);
 }
 
+/**
+ * If awaited, this delays execution for a set time using the given Scene's
+ * internal timer.
+ * @param time - The time (ms) to delay execution
```

Comment:
```suggestion
 * If awaited, this delays execution for a set time using the given Scene's internal timer.
 * @param time - The time (ms) to delay execution. Use {@linkcode fixedNumber}
 *   on the value to prevent the game speed from affecting the duration of the delay.
```
Requires a TSDoc import to be added as well.